client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -89,7 +89,7 @@ class SelectIpsTag extends Component {
 		let selectedRegistrar = this.getRegistrarInfo( ipsTagInput, ipsTagList );
 
 		if ( isEmpty( selectedRegistrar ) ) {
-			selectedRegistrar = { tag: toUpper( ipsTagInput ), registrarName: '', registrarUrl: '' };
+			selectedRegistrar = { tag: ipsTagInput.toUpperCase(), registrarName: '', registrarUrl: '' };
```

Comment:
It's a bit hard to tell, but I'm not confident that `ipsTagInput` will always be a string. I suggest doing:

```suggestion
			selectedRegistrar = { tag: ipsTagInput && ipsTagInput.toUpperCase(), registrarName: '', registrarUrl: '' };
```

Reply:
I've verified it's safe - it defaults to an empty string, and all the cases when it's modified it still remains a string (even if it's empty). I'll pass here if you don't mind 😉

Reply:
Sounds like the right call! 👍